File: Python/Empleados.py
class Empleados():
    __nombreEmpleados = []
    __sueldoEmpleados = []

    def registrarEmpleado(self, nombre, sueldo):
        self.__nombreEmpleados.append(nombre)
        self.__sueldoEmpleados.append(sueldo)

    def verificarRegistrosEmpleados(self):
        for i in range(len(self.__nombreEmpleados)):
            print("Empleado #", i+1)
            print(
                "Nombre: ", self.__nombreEmpleados[i], " sueldo: ", self.__sueldoEmpleados[i])

    def verificarCantidadEmpleados(self):
        print(len("Cantidad de empleados: ", self.__nombreEmpleados))


def main():
    print("---Programa para registro y verificacion de empleados---")
    bandera = True
    misEmpleados = Empleados()
    while bandera:
        opcion = int(input("""
    Ingresa el numero de una de las siguientes opciones
    1-Dar de alta un empleado
    2-Consulta la informacion de todos tus empleados
    3-Consulta cuantos empleados estan registrados
    4-Salir de sistema
    : """))
        if opcion == 1:
            nombre = input("Ingresa el nombre del empleado: ")
            sueldo = input("Ingresa el sueldo del empleado: ")
            misEmpleados.registrarEmpleado(nombre, sueldo)
        elif opcion == 2:
            misEmpleados.verificarRegistrosEmpleados()
        elif opcion == 3:
            misEmpleados.verificarCantidadEmpleados()
        elif opcion == 4:
            break
        else:
            print("Opcion incorrecta")
            break


if __name__ == '__main__':
    main()

File: Java/Empleados.java
import java.util.Scanner;

public class Empleados {
    private String nombreEmpleado;
    private float sueldoEmpleado;

    public Empleados(String nombre, float sueldo) {
        this.nombreEmpleado = nombre;
        this.sueldoEmpleado = sueldo;
    }

    public String getNombre() {
        return nombreEmpleado;
    }

    public float getSueldo() {
        return sueldoEmpleado;
    }

    public static void main(String[] args) {
        System.out.println("---Programa para registro y verificacion de empleados");
        boolean bandera = true;
        Scanner scanner = new Scanner(System.in);
        BDEmpleado miBase = new BDEmpleado();
        while (bandera == true) {
            System.out.println(
                    "\nIngresa el numero de una de las siguientes opcines\n1-Dar de alta un empleado\n2-Consulta la informacin de todos tus empleados\n3-Consulta cuantos empleados estan registrados\n4-Salir del sistema\nOpcion: ");
            int opcion = scanner.nextInt();
            if (opcion == 1) {
                System.out.println("Ingresa el nombre del empleado: ");
                String nombre = scanner.next();
                System.out.println("Ingresa el sueldo del empleado: ");
                float sueldo = scanner.nextFloat();
                Empleados miEmpleado = new Empleados(nombre, sueldo);
                miBase.registrarEmpleado(miEmpleado);
            } else if (opcion == 2) {
                miBase.verificarRegistroEmpleados();
            } else if (opcion == 3) {
                System.out.println("Cantidad de empleados: " + miBase.verificarCantidadEmpleados());
            } else if (opcion == 4) {
                break;
            } else {
                break;
            }
        }
        scanner.close();
    }
}
